src/System/Console/Traits/PrintHelpTrait.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace System\Console\Traits;
+
+use System\Console\Style\Style;
+
+trait PrintHelpTrait
+{
+    /**
+     * Print option describe using style console.
+     *
+     * @return Style
+     */
+    protected function printOption(Style $style)
+    {
+        return $this->printHelper($style, $this->option_describes);
+    }
+
+    /**
+     * Print argument describe using style console.
+     *
+     * @return Style
+     */
+    protected function printArgument(Style $style)
+    {
+        return $this->printHelper($style, $this->argument_describes);
```

Comment:
dont use `printHelper` is may same with `printOption` but in feature may deferent.